sources/packages/web/src/components/generic/StatusChipApplication.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+  <status-chip :status="chipStatus" :label="status" />
+</template>
+<script lang="ts">
+import { computed } from "vue";
+import StatusChip from "@/components/generic/StatusChip.vue";
+import { useApplication } from "@/composables";
+export default {
+  components: { StatusChip },
+  props: {
+    status: {
+      type: String,
```

Comment:
Can we have these properties with a better type definition, like below or adding a validator or both?
```
props: {
    status: {
      type: String as PropType<ApplicationStatus>,
      required: true,
    },
  },
```